fsd_utils/config/commonconfig.py

```diff
@@ -0,0 +1,144 @@
+from distutils.log import WARN
+import logging
+import os
+from pathlib import Path
+
+class CommonConfig:
+
+    FSD_LOG_LEVELS = {
+        "development": logging.DEBUG,
+        "unit_test": logging.DEBUG,
+        "dev": logging.INFO,
+        "test": logging.WARN,
+        "production": logging.ERROR
+    }
+
+    # ---------------
+    #  General App Config
+    # ---------------
+    SECRET_KEY = os.getenv("SECRET_KEY", "secret_key")
+    SESSION_COOKIE_NAME = os.getenv("SESSION_COOKIE_NAME", "session_cookie")
+    FORCE_HTTPS = True
```

Comment:
Can we make this `False` PaaS enforces this and enabling it creates problems locally.

Reply:
Can do. It's overriden in development and unit_test configs to False anyway for running locally, but if we don't need it for PaaS then we can make the default false